frontend/src/metabase/query_builder/components/dataref/ModelPane/ModelPane.styled.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from "@emotion/styled";
+import { space } from "metabase/styled-components/theme";
+import { color } from "metabase/lib/colors";
+import Icon from "metabase/components/Icon";
+
+export const ModelPaneContainer = styled.div``;
+
+export const ModelPaneDetail = styled.div`
+  color: ${color("text-medium")};
+  display: flex;
+  align-items: center;
+  padding: 0.25em ${space(1)};
+  font-weight: 700;
+  a {
```

Comment:
We consider styling of nested elements by tags an anti-pattern. Please use the reference to the styled component directly, e.g. like this https://github.com/metabase/metabase/blob/9deb8ea4039f05b91b54538357ba2daa86a3edb1/frontend/src/metabase/nav/components/AppBar/AppBarLarge.styled.tsx#L33

Reply:
Sure. Can you briefly explain why? I'm genuinely curious and new to styled components